scripts/completions.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -e
+rm -rf completions
+mkdir completions
+for sh in bash zsh fish; do
+  go run main.go completion "$sh" >"completions/goreleaser.$sh"
```

Comment:
Is this ran on the users computer? 

I'm not sure how homebrew works but if it does get ran on their machine it requires the go compiler which might not be available.

Reply:
Check .goreleaser.yml before hooks